docs/rst/manual/about/release_notes.rst

```diff
@@ -1,6 +1,120 @@
 Release Notes
 =============
 
+Version 2.6
+-----------
+
+Highlighted changes
+~~~~~~~~~~~~~~~~~~~
+
+Improvements to ERT CLI
+#######################
+The text and shell interface has been removed from ERT, but the CLI interface has gotten a upgrade and now
+included basic monitoring to show the progress of the running experiment. The CLI now also supports MDA and
+running single workflows.
+
+The see the usage and complete list of available commands, go to :doc:`../reference/running_ert`.
+
+
+Improvements to ERT GUI
+#######################
+The ERT GUI now includes help links to relevant resources and the job monitoring window now also includes
+memory usage information for each job. In addition, the output from the Eclipse job is treated like any
+other job and you can now read it from the the job monitoring window.
+
+Experimental features
+#####################
+The new iterative ensemble smoother algorithm has been added as an update algorithm.
+
```

Comment:
Should we mention the PCA work here as well or is it not included in this release?

Reply:
Never mind, the remove commit is part of the 2.6 branch in libres.